api_spec.yaml

```diff
@@ -4282,7 +4711,108 @@ paths:
           $ref: '#/components/responses/401'
         '500':
           $ref: '#/components/responses/500'
-    
+
+  /appid-<appId>/user/id:
+    get:
+      tags:
+        - Core
+      operationId: getUserId
+      description: |
+        Get user id from email or phone number
+      parameters:
+        - $ref: '#/components/parameters/api-key'
+        - $ref: '#/components/parameters/cdi-version'
+        - $ref: '#/components/parameters/userId'
+      responses:
+        '200':
+          description: The operation was successful
+          content:
+            application/json:
+              schema:
+                oneOf:
+                  - type: object
+                    properties:
+                      status:
+                        $ref: '#/components/schemas/statusOK'
+                      user:
+                        $ref: '#/components/schemas/authRecipeUser'
+                  - type: object
+                    properties:
+                      status:
+                        type: string
+                        enum: ['UNKNOWN_USER_ID_ERROR']
+        '400':
+          $ref: '#/components/responses/400'
+
+        '401':
+          $ref: '#/components/responses/401'
+
+        '404':
+          $ref: '#/components/responses/404'
+
+        '500':
+          $ref: '#/components/responses/500'
+
+  /appid-<appId>/<tenantId>/users/by-accountinfo:
+    get:
+      tags:
+        - Core
+      operationId: getUsersByAccountInfo
+      description: |
+        Get users by email, phoneNumber or thirdParty info
+      parameters:
```

Comment:
`doUnionOfAccountInfo` seems to be missing

Reply:
Done